death-clock.js

```diff
@@ -98,11 +101,11 @@ const tickTock = () => {
   }
 
   // Chrome Extension popup doesn't like emojis
+  // Making skull a link to the homepage of the site it is on
   const skull = (socialMediaPlatform !== 'none') ? '☠️' : '';
-
   target.innerHTML = `
       <div class="social-media-death-clock">
-        <div class="skull-emoji">${skull}</div>
+        <a href="` + siteurl + `" class="skull-emoji">${skull}</a>
```

Comment:
We can just use string interpolation here.

```suggestion
        <a href="${siteurl}" class="skull-emoji">${skull}</a>
```